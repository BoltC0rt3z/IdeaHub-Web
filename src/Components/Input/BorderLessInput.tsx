import React from 'react';
import { Input } from 'antd';
import { InputProps } from 'antd/lib/input';

import styles from './BorderLessInput.module.scss';

const BorderLessInput: React.FC<InputProps> = (props) => {
  const { className, ...restProps } = props;
  const classNames: string = [className, styles.input].join(' ');

  return <Input className={classNames} {...restProps} />;
};

export default BorderLessInput;
