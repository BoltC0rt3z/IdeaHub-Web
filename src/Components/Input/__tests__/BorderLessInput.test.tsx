import React from 'react';
import { mount, ReactWrapper } from 'enzyme';

import BorderLessInput from '../BorderLessInput';

describe('Tests for BorderLessInput component', () => {
  const wrapper: ReactWrapper = mount(<BorderLessInput />);

  it('Should render the borderless input component without crushing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
