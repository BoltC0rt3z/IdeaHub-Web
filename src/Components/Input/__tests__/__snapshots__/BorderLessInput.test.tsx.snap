// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for BorderLessInput component Should render the borderless input component without crushing 1`] = `ReactWrapper {}`;
